src/java.base/share/classes/jdk/internal/classfile/impl/DirectCodeBuilder.java

```diff
@@ -590,13 +595,13 @@ public void writeInvokeInterface(Opcode opcode,
         writeBytecode(opcode);
         bytecodesBufWriter.writeIndex(ref);
         bytecodesBufWriter.writeU1(count);
-        bytecodesBufWriter.writeU1(0);
+        bytecodesBufWriter.skip(1);
```

Comment:
`0` is mandated: https://docs.oracle.com/javase/specs/jvms/se22/html/jvms-6.html#jvms-6.5.invokeinterface